Sources/OktaDirectAuth/DirectAuthFlow.swift

```diff
@@ -244,12 +244,26 @@ public class DirectAuthenticationFlow: AuthenticationFlow {
         }
     }
     
+    /// Indicates the intent for the user authentication operation.
+    /// 
+    /// This value is used to toggle behavior to distinguish between sign-in authentication, password recovery / reset operations, etc.
+    public enum Intent: String, Codable {
```

Comment:
Maybe `EndUserIntent`?

Reply:
I named this `Intent` because that's the name of the API argument.  In cases like this, I'd suggest on Android it names this `DirectAuthenticationIntent` or something like that, but we can keep it to be `Intent` for Swift and Typescript since there's no naming conflict there.

This is similar to the naming for "Token.Type"...`Type` is a reserved word in Swift, so there I called it `Token.Kind`, but for other platforms where it's not a conflict, there's no point in using a less-than-optimal word.